fraqbot/example-config.yaml

```diff
@@ -6,20 +6,18 @@ connectors:
       token: <a Slack API token>
 helpEnabled: true
 legos:
-  AshokCoin:
+  Shakespeare:
```

Comment:
Instead of replacing the AshokCoin example, can you add your example?
Also, your new Lego doesn't take any kwargs, except acl, which is a builtin